### DISCUSSION TITLE
ts101 problem

### DISCUSSION THREAD

**Discussion by TheBeaverr** [Ralim/IronOS#1903] [Q&A]
hi i recently got a ts 101 and installed the ironOS, today after i used it and turned it off, i came back to use it and now there is a little number 21 on the top right side of the screen and it doesn't warm up , i tried to reset it to deafault but nothing the only thing that changed was the number that became 20, now i reinstalled ironOS and it still doesn't work.

**Comment by Ralim** [comment]
That number is the input voltage. So long as it's above 5 it will let you heat. It's the graphics showing that the top is disconnected? 

If not, if you take the tip out. Clean the contacts and put it back in, does that help?
